Xamarin.Forms.Platform.MacOS/Renderers/EntryRenderer.cs

```diff
@@ -289,5 +292,13 @@ void UpdateMaxLength()
 			if (currentControlText.Length > Element?.MaxLength)
 				Control.StringValue = currentControlText.Substring(0, Element.MaxLength);
 		}
+
+
+		void UpdateIsReadOnly()
+		{
+			Control.Editable = !Element.IsReadOnly;
+			if (Element.IsReadOnly && Control.Window?.FirstResponder == Control.CurrentEditor)
+				Control.Window?.MakeFirstResponder(null);
```

Comment:
@rmarinho I added this code on MacOS to fix #4834 so please re-review macos

Reply:
👍